suprb/optimizer/rule/mutation.py

```diff
@@ -160,3 +161,52 @@ def min_percentage(self, rule: Rule, random_state: RandomState):
         bounds = rule.match.bounds
         bounds[:, 0] -= random_state.uniform(0, self.sigma[0], size=bounds.shape[0])
         bounds[:, 1] += random_state.uniform(0, self.sigma[1], size=bounds.shape[0])
+
+
+class AdaptiveMutation(RuleMutation):
+    """Start off by using a given operator until the fitness decreased
+       more than self.number_of_worse_iterations times. Then change the mutation operator
+       to Normal
+    """
+
+    def __init__(self, matching_type: MatchingFunction = None,
+                 sigma: Union[float, np.ndarray] = 0.1,
```

Comment:
does this sigma have an effect or is it just a dummy? we could just parse `operator`'s sigma to super, couldn't we?